packages/core-components/src/layout/Sidebar/Bar.tsx

```diff
@@ -21,17 +21,25 @@ import classnames from 'classnames';
 import React, { useState, useContext, useRef } from 'react';
 import Button from '@material-ui/core/Button';
 
-import { sidebarConfig, SidebarContext } from './config';
+import {
+  makeSidebarConfig,
+  makeSidebarSubmenuConfig,
+  SidebarConfig,
+  SidebarContext,
+  SidebarConfigContext,
+  SubmenuConfig,
+  SidebarOptions,
+  SubmenuOptions,
+} from './config';
 import { BackstageTheme } from '@backstage/theme';
 import { SidebarPinStateContext, useContent } from './Page';
 import { MobileSidebar } from './MobileSidebar';
 
 /** @public */
 export type SidebarClassKey = 'drawer' | 'drawerOpen';
-
-const useStyles = makeStyles<BackstageTheme>(
+const useStyles = makeStyles<BackstageTheme, { sidebarConfig: SidebarConfig }>(
   theme => ({
-    drawer: {
+    drawer: props => ({
```

Comment:
I have overridable components overriding this css, but after this one is changed to a function, my overridable is not working and this css is missing in my backstage app and ruin the overall styling.
At the moment I am disabling the custom css, but any help on how to make this overridable again?